apps/mark-scan/frontend/src/app_states.test.tsx

```diff
@@ -99,7 +99,7 @@ test('`waiting_for_invalidated_ballot_confirmation` state renders ballot invalid
     'waiting_for_invalidated_ballot_confirmation.paper_present'
   );
   apiMock.setAuthStatusCardlessVoterLoggedInWithDefaults(electionDefinition);
-  await screen.findByText('Ask a Poll Worker for Help');
```

Comment:
I would consider "ask a poll worker for help" a bit plainer than "notify a poll worker." I know it's a matter of opinion but generally important to think about (more on this here: https://civicdesign.org/topics/plain-language/)

Reply:
Agreed - I made this change to avoid wrapping a single word to the next line in the title. Any ideas for more concise, plain language that could fit on one line? Maybe "Ask a Poll Worker" or "Poll Worker Needed"?

Reply:
Hmm, could we just shrink the font size a tiny bit or change the padding? I don't have a visual reference so if that's a bad idea, ignore me.

Reply:
[6207da7](https://github.com/votingworks/vxsuite/pull/4592/commits/6207da7bac4decbe3609ec8c5f8d8ee13b7a617a) reduces the font size